airbyte-ci/connectors/base_images/base_images/version_registry.py

```diff
@@ -0,0 +1,260 @@
+#
```

Comment:
💅  My single nit pick!

We have alot of files, not a lot of folders.

Folders are a great way to share the taxonomy that you had in your head when developing with a developer first learning the code base.

I dont think we should go overboard but in my head I see a few separate components

1. Root images (sub divided by language)
2. Base images (sub divided by abstract implementation and language)
3. Version Registry (including published images)
4. Utilities (catch all)
5. CI 

So could we move 
- version_registry.py
- publish.py
- and perhaps published_image.py

To a shared folder?

A few other (very very optional) ideas are
* place commands.py in a folder called ci
* use a folder structure to show a relationship bewteen the bases.py 
    * `airbyte-ci/connectors/base_images/base_images/images/abstract.py`
    * `airbyte-ci/connectors/base_images/base_images/images/python/base.py`
    * and eventually `airbyte-ci/connectors/base_images/base_images/images/java/base.py`